lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json

```diff
@@ -6782,9 +6782,9 @@
       "occupancy_activity_schedule": "HotelLarge ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-2007",
-      "ventilation_primary_space_type": "General",
-      "ventilation_secondary_space_type": "Corridors",
-      "ventilation_per_area": 0.06,
+      "ventilation_primary_space_type": "Public Spaces",
+      "ventilation_secondary_space_type": "Corridors and utilities",
+      "ventilation_per_area": 0.05,
```

Comment:
The primary and secondary space types were corrected and the ventilation rates were updated accordingly.